astropy/io/votable/tests/test_vo.py

```diff
@@ -276,7 +279,57 @@ def test_select_columns_by_name():
     assert array["string_test"][0] == "String & test"
     for c in columns:
         assert not np.all(mask[c])
-    assert np.all(mask["unicode_test"])
+
+    # deselected columns shouldn't be present in the output
+    assert "unicode_test" not in array.dtype.fields
+    assert "unicode_test" not in mask.dtype.fields
+
+
+@pytest.mark.parametrize(
+    "column_ids, use_names_over_ids, expected_names",
+    [
+        # just the first column
+        (["string_test"], False, ["string_test"]),
```

Comment:
The separate list for `ids` could be avoided:
```suggestion
        pytest.param(["string_test"], False, ["string_test"], id="first-col-ids"),
```